SmartPlaces.Facilities/samples/Topology/README.md

```diff
@@ -49,3 +49,102 @@ The following secrets must be defined in the KeyVault in order to run this code:
 ## Logging and Metrics
 
 Logs and Metrics will appear in the Application Insights instance pointed to by the App Insights connection string. For more information on how to configure the Application Insights SDK, please see: https://learn.microsoft.com/en-us/azure/azure-monitor/app/worker-service
```

Comment:
Should we add something around the verbosity of these logs are set in the [appsettings.json](https://github.com/Azure/opendigitaltwins-tools/blob/main/SmartPlaces.Facilities/samples/Topology/src/appsettings.json)? The topology sample is a bit light, but the Telemetry and EdgeGateway ones output signs of life to the console and full trace logs to appInsights.